package.json

```diff
@@ -13,6 +13,11 @@
 			"name": "Daniel Hillmann",
 			"url": "https://github.com/hilleer",
 			"email": "hiller@live.dk"
+		},
+		{
+			"name": "Michael Stocks",
+			"url": "https://github.com/Droxx",
+			"email": "michaelstocks1@gmail.com"
```

Comment:
This is unnecessary - I believe you'll automatically be marked as a contributor for the repository. This field, as I understand, is more meant as active maintainers of the repository - e.g. those that have write/admin access.

Reply:
Understood! This is the first time I've forked existing work so I was unsure of best practices